cdserver/README.md

```diff
@@ -16,6 +16,6 @@ Run `rails db:setup` and `rails db:migrate` to create and set up database
 Run `rails server`, navigate to `localhost:3000`
 Run `rails routes` to view endpoints
 
-## Running Unit tests
+## Running tests
 
-Run `rspec spec` to run all unit tests
```

Comment:
We will want *both* unit tests and cucumber tests.    We will start with cucumber tests, and only add unit tests when needed to hit boundary conditions or whatever to get to 100% code coverage.  See this article for a fantastic explanation: http://gasparnagy.com/2017/02/balancing-scenarios-and-unit-tests-case-study-with-specflow/  (specflow is the .NET version of cucumber, same concept)